aws/src/test/java/org/apache/iceberg/aws/s3/TestVendedCredentialsProvider.java

```diff
@@ -26,6 +26,7 @@
 
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import org.apache.iceberg.CatalogProperties;
```

Comment:
tests still need to be updated. See also my comment in https://github.com/apache/iceberg/pull/12612#discussion_r2018013789